recipe/meta.yaml

```diff
@@ -1,17 +1,26 @@
-{% set version = "3.0.0b3" %}
-{% set version_str = "v3.0.0-beta-3" %}
+{% set name = "protobuf" %}
+{% set version = "3.0.0" %}
+{% set version_str = "v3.0.0" %}
+{% set sha256 = "f5b3563f118f1d3d6e001705fa7082e8fc3bda50038ac3dff787650795734146" %}
```

Comment:
Out of curiosity, why are we templating the hash? Is that *ever* used in multiple places?